package.json

```diff
@@ -49,7 +50,10 @@
     "@storybook/preset-create-react-app": "^2.1.1",
     "@storybook/react": "^5.3.17",
     "autoprefixer": "^9.7.5",
+    "commitizen": "^4.1.2",
     "concurrently": "^5.1.0",
+    "cz-conventional-changelog": "3.2.0",
+    "husky": "^4.2.5",
```

Comment:
@genzographics between these packages I was able to have an npm tool fire on `git commit` and help me write a conventional commit:

![image](https://user-images.githubusercontent.com/2053583/87223873-71258e80-c378-11ea-90ec-419fed253e27.png)

![image](https://user-images.githubusercontent.com/2053583/87223881-8a2e3f80-c378-11ea-8c76-a1e0f65af4aa.png)

[cz-cli](https://github.com/commitizen/cz-cli)
[husky](https://github.com/typicode/husky)